### DISCUSSION TITLE
Infinity-scroll?

### DISCUSSION THREAD

**Discussion by dnh33** [Norserium/react-indiana-drag-scroll#64] [Q&A]
Is there a way to add infinity-scrolling to the container?

**Comment by Norserium** [comment]
@dnh33, frankly, I'm not sure. But if you set `overflow: auto !important;` it maybe will work. In any-case, it's still the scrollable container.

Try to add the infinity scrolling to a simple container, then when you get the desired result, replace your container by the one from this library.
